keyboards/handwired/wwa/rosetta/info.json

```
@@ -0,0 +1,47 @@
+{
+    "manufacturer": "spacehangover",
+    "keyboard_name": "Rosetta",
+    "maintainer": "spacehangover",
+    "development_board": "promicro",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
```

migrated from `rules.mk`
```suggestion
        "nkro": true,
        "midi": true
    },
    "build": {
       "lto": true
    },
```